generated/Endpoint/ChatUpdate.php

```diff
@@ -66,16 +66,16 @@ public function getExtraHeaders(): array
     protected function getFormOptionsResolver(): \Symfony\Component\OptionsResolver\OptionsResolver
     {
         $optionsResolver = parent::getFormOptionsResolver();
-        $optionsResolver->setDefined(['blocks', 'attachments', 'text', 'ts', 'parse', 'as_user', 'link_names', 'channel']);
-        $optionsResolver->setRequired([]);
+        $optionsResolver->setDefined(['blocks', 'attachments', 'as_user', 'ts', 'parse', 'text', 'link_names', 'channel']);
+        $optionsResolver->setRequired(['ts', 'channel']);
         $optionsResolver->setDefaults([]);
         $optionsResolver->setAllowedTypes('blocks', ['string']);
         $optionsResolver->setAllowedTypes('attachments', ['string']);
-        $optionsResolver->setAllowedTypes('text', ['string']);
+        $optionsResolver->setAllowedTypes('as_user', ['string']);
```

Comment:
I don't understand why Slack changed this boolean to a string... But let's see :grimacing: